packages/reapit-connect-management/src/components/ui/users/user-groups-tab.tsx

```diff
@@ -50,7 +54,18 @@ const UserGroupsTab: React.FC = () => {
           each group by selecting ‘Manage’.
         </i>
       </Section>
-      {!data ? <Loader /> : <UserGroupsContent data={data} columns={columns} onPageChange={onPageChange} />}
+      {!data ? (
```

Comment:
This will hang on the loader if data fetching fails. Needs a to be tied to a loading state

Reply:
Ignore this, I get it